### ISSUE TITLE
Moving Borderless Window

### ISSUE THREAD

**Issue by primesoftware** [Kode/Kha#645]
How do you move a borderless window programmatically? I am compiling for desktop systems. I saw you can set the x and y when you create the window using initEx, but what about after window creation?

**Comment by malublu** [comment]
Simple try ::SetWindowPos(_hwnd , NULL, x, y, w, h, SWP_NOZORDER | SWP_NOACTIVATE); for windows target.
For Linux you must look in the x11 api.
And on mac ... i don't know ^^

But i hope i can help you with that information.

**Comment by sh-dave** [comment]
It's currently not possible, but is there an actual usecase for it?

**Comment by malublu** [comment]
I think: year. You can make games like windowframe (itch.io) :grinning:

**Comment by primesoftware** [comment]
@sh-dave I am writing a program to do some 3D molecular drawing, but need GUI widgets for the non-3D stuff. My plan was to go JavaFX for the main app, and then kick off a process for the 3D input. Then I had the hair brained idea of trying to center the 3D input screen (Kha based) over a JavaFX widget to give it the appearance that its one and the same. The problem is if the Java window got moved I would need to move the Kha window too. In any case, I've since abandoned that terrible idea lol!
